### PR TITLE
New post notifications

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: css/layout.less [hunk 1]
```diff
@@ -602,32 +602,48 @@ nav {
...
     position: fixed;
     left: 50%;
     margin-left: -@refresh-button-width/2;
-    top: 0;
+    top: @top-nav-height + 10;
     margin-top: 5px;
   }
+
+  &:after {
+  	bottom: 100%;
```
Comment (levity) [inline css/layout.less]: indentation's off

File: src/components/RefreshButton.js [hunk 2]
```diff
@@ -2,13 +2,18 @@ import React from 'react'
 import { throttle } from 'lodash'
 import { VelocityTransitionGroup } from 'velocity-react'
 import { position, viewportTop } from '../util/scrolling'
+import { nounCount } from '../util/text'
 import cx from 'classnames'
-const { func } = React.PropTypes
+const { func, number } = React.PropTypes
+
+const topNavHeight = 75
```
Comment (levity) [inline src/components/RefreshButton.js]: this varies on mobile

File: src/containers/ConnectedPostList.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ export class ConnectedPostList extends React.Component {
       clearInterval(this.intervalId)
     }
 
-    this.intervalId = setInterval(dispatchCheckFreshness, 60 * 1000)
+    this.intervalId = setInterval(dispatchCheckFreshness, 15 * 1000)
```
Comment (levity) [inline src/containers/ConnectedPostList.js]: i'm assuming you changed this for testing but forgot to change it back?

File: src/reducers/util.js [hunk 4]
```diff
@@ -76,12 +76,12 @@ export const keyedCounter = (actionType, payloadKey, statePath = 'meta.cache.id'
     return state
   }
 
-export const keyedHasFreshItems = (actionType, bucket) =>
+export const keyedCountFreshItems = (actionType, bucket) =>
```
Comment (levity) [inline src/reducers/util.js]: hmm. if we're going to encourage reuse of this, maybe we can just call it `keyedCount` and have the key in the payload be `count`

File: test/client/components/ConnectedPostList.test.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      pending: {},
+      countFreshPostsByQuery: {
+        'subject=community&id=foo': 3
+      }
+    }
+
+    it('displays new post notification', () => {
+      const props = {subject: 'community', id: 'foo'}
+      const store = mocks.redux.store(state)
+      const context = {store, dispatch: store.dispatch, community}
```
Comment (levity) [inline test/client/components/ConnectedPostList.test.js]: you don't need to have `dispatch` here unless you want a custom spy; `createElement` automatically sets it

**Comment by robbiecarlton** [comment]
made those changes

**Comment by levity** [comment]
:+1:

**Pull request merged**